docs/sources/examples/rss-feeds.md

```diff
@@ -22,29 +22,29 @@ weight: 8003
 
 # Visualizing data from AWS Status feeds
 
-In this example, we are going to see how we can use AWS status feeds as Grafana annotations. This will be useful when you are dealing with any AWS outages. For this, we are going to use the [AWS RSS feed](https://status.aws.amazon.com/rss/all.rss). You can find more feeds at [AWS Status page](https://status.aws.amazon.com/)
+In this example, we are going to see how we can use AWS status feeds as Grafana annotations. This can be be useful when you are dealing with any AWS outages. For this, we are going to use the [AWS RSS feed](https://status.aws.amazon.com/rss/all.rss). You can find more feeds at [AWS Status page](https://status.aws.amazon.com/)
```

Comment:
https://developers.google.com/style/person

```suggestion
In this example, you use AWS status feeds as Grafana annotations.
This can be be useful when you are dealing with any AWS outages.
For this, you're going to use the [AWS RSS feed](https://status.aws.amazon.com/rss/all.rss).
You can find more feeds at [AWS Status page](https://status.aws.amazon.com/)
```